### PR TITLE
Update supported log types with recent logs

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Fixes issue #5145

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security-analytics/sec-analytics-config/log-types.md [hunk 1]
```diff
@@ -14,28 +14,79 @@ To navigate to the **Log types** page, select **Log types** under **Detectors**
...
+
+| Log type | Description | 
+|:--- |:--- |
+| `waf` | Web Application Firewall (WAF) logs (introduced in OpenSearch 2.11) for users that require monitoring of the WAF use case that's provided out of the box with Security Analytics. The role of WAF is to monitor and filter HTTP traffic between a web application and the internet. WAF prevents common security attacks, such as cross-site scripting (XSS) and SQL Injection (SQi). |
+| `linux` | Linux system logs that record Linux syslog events. |
+| `zeek` | Zeek monitoring logs created when Zeek detects new network activity. |
+| `cisco` | Cisco logs for when a user tries to authenticate to a [Cisco router or firewall](https://www.cisco.com/c/en/us/support/docs/security-vpn/terminal-access-controller-access-control-system-tacacs-/10384-security.html#anc1).
+
+
+### System acivity
```
Comment (Naarcha-AWS) [inline _security-analytics/sec-analytics-config/log-types.md:64]:
```suggestion
### System activity
```

**Comment by eirsep** [comment]
thanks for making this change @Naarcha-AWS 
It would be very valuable for us to actually mention what are the field mappings for each log type that we use in our rule names (these are static configurations found in the plugin repo on Opensearch-Project)
That way users who have divergence in their mappings can be guided on how to using security analytics create mappings api to enable creation of detectors based on the rules they create.

**Comment by Naarcha-AWS** [comment]
Amardeepsingh is going to help me finalize this PR with some updated content. Keeping this open, but its on hold for now.

**Comment by Naarcha-AWS** [comment]
Closing this in favor of #6235

**Pull request closed without merging**